packages/react-ui/src/components/molecules/Autocomplete.js

```diff
@@ -0,0 +1,93 @@
+import React, { forwardRef } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Divider,
+  ListItemIcon,
+  ListItemText,
+  Autocomplete as MuiAutocomplete,
+  createFilterOptions
+} from '@mui/material';
+import { AddCircleOutlineOutlined } from '@mui/icons-material';
+import MenuItem from './MenuItem';
+
+const filter = createFilterOptions();
+
+const Autocomplete = forwardRef(
+  (
+    {
+      creatable,
+      newItemTitle,
+      freeSolo,
+      renderOption,
+      forcePopupIcon,
+      filterOptions,
+      getOptionLabel,
+      ...otherProps
+    },
+    ref
+  ) => {
+    // forwardRef needed to be able to hold a reference, in this way it can be a child for some Mui components, like Tooltip
+    // https://mui.com/material-ui/guides/composition/#caveat-with-refs
+
+    const creatableOptions = (options, params) => {
+      const filtered = filter(options, params);
+      const { inputValue } = params;
+
+      const isExisting = options.some((option) => inputValue === option.title);
+
+      if (inputValue.length > 1 && inputValue !== '' && !isExisting) {
+        filtered.push({
+          inputValue,
+          title: newItemTitle || `Add "${inputValue}"`
```

Comment:
'en' file resource level for the 'Add' ?

Reply:
Done https://github.com/CartoDB/carto-react/pull/873/commits/1bdfbda8be775305e7b1d211733ba27b2eb0871b